apps/laboratory/src/pages/library/multichain-basic.tsx

```diff
@@ -26,14 +26,16 @@ const modal = createAppKit({
   networks: networks as [AppKitNetwork, ...AppKitNetwork[]],
   defaultNetwork: mainnet,
   projectId: ConstantsUtil.ProjectId,
-  metadata: ConstantsUtil.Metadata
+  metadata: ConstantsUtil.Metadata,
+  basic: true
```

Comment:
this is no longer needed, will refactor this page completely into different sub-pages.
@svenvoskamp could we maybe use your examples to build pages to test on lab? Or do you think they should be separate? 

Could you link me to some code whenever available?